scripts/shell/es_delete_alias.sh

```diff
@@ -0,0 +1,6 @@
+#!/usr/bin/env bash
+
+NAME=$1
+URL="localhost:9200/_all/_alias/"$NAME
```

Comment:
Syntax note according to the style guide I've been following (https://google.github.io/styleguide/shell.xml#Variable_expansion). There are cases where one can't use quotes+braces, but pretty rare. `$1` is a single character positional parameter, so doesn't use quotes+braces, but I think these lines would be:

```bash
URL="localhost:9200/_all/_alias/${NAME}"

curl -XDELETE "${URL}?pretty"
```